apps/style/applab/style.scss

```diff
@@ -555,12 +555,21 @@ div#visualizationResizeBar {
       }
     }
   }
+  .Select-value-label .checkbox {
+    visibility: hidden;
+  }
   .Select-menu, .Select-menu-outer {
-    height: 516px;
-    max-height: 516px;
+    height: 75vh;
+    max-height: 75vh;
```

Comment:
We don't use viewport units too often.  I'm curious if there will be any issues, particularly on mobile devices, but I think this seems safe, and at 75% we avoid any of the tricky issues that some developers have faced, especially when they care about the [definition of 100%](https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser).

Reply:
I tried it on chrome dev tools for the iPhone X and Pixel 2 and decided it looked better at 70vh where it doesnt cut off. Using percentages didn't work with this component and I didn't want to use an exact pixel count to extend the dropdown in case that went longer than the page.
<img width="865" alt="Screen Shot 2020-11-10 at 2 22 04 PM" src="https://user-images.githubusercontent.com/17147070/98740574-1e78d500-2360-11eb-9e16-c23e6b0de828.png">
<img width="865" alt="Screen Shot 2020-11-10 at 2 21 39 PM" src="https://user-images.githubusercontent.com/17147070/98740649-4405de80-2360-11eb-9cfe-85898e27efeb.png">